src/plugin.css

```diff
@@ -1,77 +1,45 @@
 /* Sass for videojs-seek-buttons */
 
-/* Using material icon font */
-@import url(https://fonts.googleapis.com/css?family=Material+Icons);
-
 .video-js {
 
   /* This class is added to the video.js element by the plugin by default. */
   & .vjs-seek-button {
-    font-family: 'Material Icons';
+    font-family: 'VideoJS';
     cursor: pointer;
+    font-weight: 400;
+    font-style: normal;
 
     &.skip-back::before {
-      content: '\e042';
-    }
-    &.skip-back.skip-10::before {
-      content: '\e059';
-    }
-    &.skip-back.skip-5::before {
-      content: '\e05b';
-    }
-    &.skip-back.skip-30::before {
-      content: '\e05a';
+      transform: rotate(-45deg);
+      -ms-transform: rotate(-45deg); /* IE 9 */
+      -webkit-transform: rotate(-45deg); /* Safari */
```

Comment:
Unfortunately I can't  test this on Safari or IE.

Reply:
IE9 and Safari 8 are beyond ancient now anyway